frontend/views/containers/payments/PaymentNextDistributionPill.vue

```diff
@@ -18,8 +18,13 @@ export default ({
     ]),
     nextDistributionDateShort () {
       const nextDisDate = this.dueDateForPeriod(this.currentPaymentPeriod)
-
-      return humanDate(nextDisDate, { month: 'short', day: 'numeric' })
+      const localeDateTimeStrings = humanDate(nextDisDate, {
+        month: 'short',
+        day: 'numeric',
+        hour: '2-digit',
+        minute: '2-digit'
+      }).split(', ')
+      return `${localeDateTimeStrings[1]} - ${localeDateTimeStrings[0]}`
```

Comment:
We cannot assume that the string will have a `,` that can be split like this. Please just return the `humanDate` directly, without this ` - `